tracks/python/exercises/say/mentoring.md

```diff
@@ -72,16 +72,16 @@ Next, the instructions suggest beginning by creating a function which works for
 The return value of this function will typically be a string, but it can also be a list of strings for later assembly.
 Whether in this function or for the final return, encourage them to use a list and combine the elements using `str.join`.
 
-The main challenges are ensuring that the 'hundred' denominator is conditionally included and that the tens and ones columns are joined with a hyphen if they are both present.
+The main challenges are ensuring that the 'hundred' denominator is conditionally included and that the tens and one's columns are joined with a hyphen if they are both present.
```

Comment:
Similar to "tens", this can be read as "the quantity of base one values". In which case, this is not a possessive term and the spelling is correct as-is.